src/schema/v2/partner.ts

```diff
@@ -47,6 +47,11 @@ const artworksArgs: GraphQLFieldConfigArgumentMap = {
     type: GraphQLInt,
     description: "Return artworks with less than x additional_images.",
   },
+  missingPriorityMetadata: {
+    type: GraphQLBoolean,
+    description:
+      "Returns artworks with less than 2 images and/or missing visible price info",
```

Comment:
Maybe we should have the description be a little more general in case we want to add more missing metadata fields to this in the future!

I'll do the same in the [Gravity PR](https://github.com/artsy/gravity/pull/13851/files#diff-fd09c85dae04ac24f2ec65ca507d7f9849111b18f8c741b287abd98fe8fe0dc2R112) :]

Reply:
Oh yeah great call!